src/PrimerHeader.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react'
+import {Link} from '@primer/components'
+import Header from './Header'
+import NavDropdown from './NavDropdown'
+import NavItem from './NavItem'
+
+const PrimerHeader = ({root}) => {
```

Comment:
PrimerHeader includes all the links specific to Primer docs sites, while Header is a flexible header component that can work for any docs site